mvp/src/main/java/com/distillery/android/blueprints/mvp/architecture/BasePresenter.kt

```diff
@@ -0,0 +1,5 @@
+package com.distillery.android.blueprints.mvp.architecture
+
+import org.koin.core.KoinComponent
+
+abstract class BasePresenter(private val contractView: BaseContractView?) : KoinComponent, Presentable
```

Comment:
Same here - the word `contract` is useless
```suggestion
abstract class BasePresenter(private val view: BaseContractView?) : KoinComponent, Presentable
```